### ISSUE TITLE
Fix broken and missing links in En, Jp, zh-Hans contributior_docs web_accessibility.md

### ISSUE THREAD

**Issue by 22nds** [processing/p5.js#7204]
### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build process
- [ ] Unit testing
- [ ] Internationalization
- [ ] Friendly errors
- [X] Other (specify if possible)

### p5.js version

latest

### Web browser and version

all

### Operating system

all

### Steps:
1. Go to https://github.com/processing/p5.js/blob/main/contributor_docs/web_accessibility.md
2.  Missing/incorrect links are in the second and third paragraph 
"If you want to make your sketches [screen reader](https://en.wikipedia.org/wiki/Screen_reader)-accessible, visit the How to label your p5.js code tutorial.

If you want to use p5.js with a screen reader, visit the [Using p5.js with a Screen Reader tutorial](https://p5js.org/learn/p5-screen-reader.html)."

### Corrections for English version: 
- Missing link How to label your p5.js code. Correct link is https://p5js.org/tutorials/writing-accessible-canvas-descriptions/
- Link to Using p5.js with a Screen Reader tutorial doesn't work. Correct link is https://p5js.org/tutorials/p5js-with-screen-reader/

Affected docs are in the following languages: English, JP, zh-Hans

Please assign this issue to me and I'll submit PR.

**Comment by williamthazard** [comment]
Hi! Just wanted to check in on this since it's been a couple days. Is there anything else that needs to be done on filing this issue, per the bot? Or is there any other way I can help with this? This seems like a pretty easy fix; it'd be great to see it resolved and to see accessibility consequently improved

**Comment by Qianqianye** [comment]
Thanks @22nds for opening this issue, and @williamthazard for your interests in working on this. There are multiple broken links issues across p5.js repo and p5.js website repo, so it's taking us a bit longer to coordinate. For the particular links mentioned in this issue, they are fixed in the [English](https://github.com/processing/p5.js/blob/main/contributor_docs/web_accessibility.md) version. 

If you are interested, please feel free to open a PR to fix the links in [ja](https://github.com/processing/p5.js/blob/main/contributor_docs/ja/web_accessibility.md) and [zh-hans](https://github.com/processing/p5.js/blob/main/contributor_docs/zh-Hans/web_accessibility.md) translated version. Thank you!

**Comment by Bhavyabhardwaj** [comment]
This issue is still opened. Please assign this issue to me and I'll submit PR.

**Comment by koushikcs562** [comment]
Hello @Qianqianye and maintainers,

I see that the fix is still needed for the Japanese (`ja`) and Simplified Chinese (`zh-hans`) translated files, as noted in your comment on September 13th.

I would like to take this on. I will locate the respective translated files for `contributor_docs/web_accessibility.md` and update the two broken links using the correct URLs provided in the issue description.

Since the scope is clearly defined for documentation updates, I can likely submit a Pull Request within the next 24 hours.

Please assign this issue to me. Thank you!
